app/models/solidus_content/entry_type.rb

```diff
@@ -2,6 +2,8 @@ class SolidusContent::EntryType < ActiveRecord::Base
   has_many :entries
   after_initialize { self.options ||= {} }
 
+  scope :by_type_name, ->(type_name) { find_by!(name: type_name) }
```

Comment:
I think `by_name` would be as clear, plus shorter, any reason we should avoid it?